practice/src/scripts/services/localStorage.ts

```diff
@@ -0,0 +1,29 @@
+import handleError from "../helpers/messages";
+import { Task } from "./types";
+
+interface Storage {
+    getTasks: () => Task[];
+    saveTasks: (tasks: Task[]) => void;
+}
+
+const storage: Storage = {
+    getTasks: () => {
+        try {
```

Comment:
should cache result and return one time.
Example:
let task = [];
try {
const dataStoraged = localStorage.getItem('tasks');
tasks = dataStoraged ? JSON.parse(dataStoraged) : [];
} catch (error) {
            customErrorMessages('getting tasks', error);
        }

return tasks;